BasicSample/Assets/Shared/Scripts/RuntimeInfo.cs

```diff
@@ -15,18 +17,30 @@ public class RuntimeInfo : MonoBehaviour
         [SerializeField]
         private TextMesh runtimeText = null;
 
+        [SerializeField]
+        private ARSession arSession = null;
+
+        private void Start()
+        {
+            arSession = FindObjectOfType<ARSession>();
```

Comment:
We should probably null check this before reassigning it, since the field above is marked as serialized and might be assigned in the editor before the app is run. Otherwise, this wipes away the inspector assignment.

If we don't want this assigned early, we could also remove the serialization tag.

Reply:
Is this needed since we are doing FindObjectByType and not initializing by ourselves?